.circleci/config.yml

```diff
@@ -177,9 +229,76 @@ jobs:
           path: /tmp/resource-usage
           destination: resource-usage
 
+  unit-test-windows:
+    executor:
+      name: win/default
+      shell: bash.exe
+    steps:
+      - dependencies-windows
+
+      # - run:
+      #     name: Memory Monitor
+      #     command: |
+      #       mkdir /tmp/resource-usage
+      #       export FILE=/tmp/resource-usage/memory.txt
+      #       while true; do
+      #         ps -u root eo pid,%cpu,%mem,args,uname --sort=-%mem >> $FILE
+      #         echo "----------" >> $FILE
+      #         sleep 1
+      #       done
+      #     background: true
+
+      # Note: The setup.py checksum in the cache name ensures that we retrain our expert
+      # policies, which are cached in the pytest cache, whenever setup.py changes. This
+      # is just a rough heuristic to decide when the experts should be retrained but
+      # still better than nothing.
+      - restore_cache:
+          keys:
+            - v8-win-pytest-cache-{{ checksum "setup.py" }}
```

Comment:
Does this need to be Windows specific? Expert policies should be saved in same format regardless of underlying OS.

If this does need to be separate, then should change other one to start `v8-unix-pytest-cache-` for symmetry.

Reply:
I wasn't sure if it would be OS-specific or not, so I decided to cache it separately for now.
I can remove it.

Reply:
It shouldn't be OS specific, so let's use the same keys and can revert if something breaks. (If it does break we should understand why... as we want our policies to be shareable across platforms. But that can be a new PR.)